src/XrdCeph/XrdCephBulkAioRead.cc

```diff
@@ -0,0 +1,237 @@
+#include "XrdCephBulkAioRead.hh"
+
+bulkAioRead::bulkAioRead(librados::IoCtx *ct, logfunc_pointer logwrapper, std::string filename, size_t object_sz) {
+  /**
+   * Constructor.
+   *
+   * @param ct                Rados IoContext object
+   * @param logfunc_pointer   Pointer to the function that will be used for logging
+   * @param filename          Name of the file (not ceph object!) that is supposed to be read
+   * @param object_sz         Maximum size of the ceph objects that the file is split into
+   *
+   */
+  context = ct;
+  object_size = object_sz;
```

Comment:
Sanity-check `object_sz`?

Reply:
`CephFileRef` is used now, so sanity check shoud be done in its constructor, I think.